test/e2e/scheduling/ubernetes_lite.go

```diff
@@ -52,19 +56,33 @@ var _ = SIGDescribe("Multi-AZ Clusters", func() {
 		msg := fmt.Sprintf("Zone count is %d, only run for multi-zone clusters, skipping test", zoneCount)
 		e2eskipper.SkipUnlessAtLeast(zoneCount, 2, msg)
 		// TODO: SkipUnlessDefaultScheduler() // Non-default schedulers might not spread
+
+		cs = f.ClientSet
+		ns = f.Namespace.Name
+		nodeList = &v1.NodeList{}
+		e2enode.WaitForTotalHealthy(cs, time.Minute)
+		nodeList, err = e2enode.GetReadySchedulableNodes(cs)
+		if err != nil {
+			framework.Logf("Unexpected error occurred: %v", err)
+		}
```

Comment:
We can use framework.ExpectNoError(err) instead of line 65-67.

Reply:
done